.changeset/strong-stingrays-provide.md

```diff
@@ -0,0 +1,9 @@
+---
+'@astrojs/solid-js': patch
+'@astrojs/vue': patch
+'@astrojs/svelte': patch
+'@astrojs/react': patch
+'@astrojs/preact': patch
+---
+
+Resolve `vite` peer dependency problem in the integration packages for strict package managers like **Yarn in PnP mode**.
```

Comment:
```suggestion
Resolve `vite` peer dependency problem for strict package managers like **Yarn in PnP mode**.
```

The changesets here are generated and reused for each integration's changelogs, so I don't think we need to mention about the integration packages here as a whole.